docker/Dockerfile_pretrained_embeddings_spacy_en

```diff
@@ -1,87 +1,61 @@
-# Create common base stage
-FROM python:3.6-slim as base
-
-WORKDIR /build
-
-# Create virtualenv to isolate builds
-RUN python -m venv /build
-
-# Install common libraries
-RUN apt-get update -qq \
- && apt-get install -y --no-install-recommends \
-    # required by psycopg2 at build and runtime
-    libpq-dev \
-     # required for health check
-    curl \
- && apt-get autoremove -y
-
-# Make sure we use the virtualenv
-ENV PATH="/build/bin:$PATH"
-
-# Stage to build and install everything
-FROM base as builder
-
-WORKDIR /src
-
-# Install all required build libraries
-RUN apt-get update -qq \
- && apt-get install -y --no-install-recommends \
-    build-essential \
-    wget \
-    openssh-client \
-    graphviz-dev \
-    pkg-config \
-    git-core \
-    openssl \
-    libssl-dev \
-    libffi6 \
-    libffi-dev \
-    libpng-dev
-
-# Make sure we have the latest pip version
-RUN pip install -U pip
-
-# Copy only what we really need
-COPY README.md .
-COPY setup.py .
-COPY setup.cfg .
-COPY MANIFEST.in .
-COPY alt_requirements/ ./alt_requirements
-COPY requirements.txt .
-COPY LICENSE.txt .
-
-# Install dependencies
-RUN pip install --no-cache-dir -r alt_requirements/requirements_pretrained_embeddings_spacy.txt
-
-# Install and link spacy model
-RUN pip install https://github.com/explosion/spacy-models/releases/download/en_core_web_md-2.1.0/en_core_web_md-2.1.0.tar.gz#egg=en_core_web_md==2.1.0 --no-cache-dir > /dev/null \
-    && python -m spacy link en_core_web_md en
-
-# Install Rasa as package
-COPY rasa ./rasa
-RUN pip install .[sql,spacy]
-
-# Runtime stage which uses the virtualenv which we built in the previous stage
-FROM base AS runner
-
-WORKDIR /app
-
-# Copy over default pipeline config
-COPY docker/configs/config_pretrained_embeddings_spacy_en.yml config.yml
-
-# Copy virtualenv from previous stage
-COPY --from=builder /build /build
+FROM python:3.6-slim as python_builder
+RUN apt-get update -qq && \
+  apt-get install -y --no-install-recommends \
+  build-essential \
+  wget \
+  openssh-client \
+  graphviz-dev \
+  pkg-config \
+  git-core \
+  openssl \
+  libssl-dev \
+  libffi6 \
+  libffi-dev \
+  libpng-dev \
+  # required by psycopg2 at build and runtime
+  libpq-dev \
+  curl
+
+# install poetry
+# keep this in sync with the version in pyproject.toml and Dockerfile
+ENV POETRY_VERSION 1.0.3
```

Comment:
could we make the POETRY_VERSION a docker build time variable using `ARG` and use the latest if none is specified? looking for a way to avoid repeating this in all the dockerfiles

Reply:
I think we decided to work on this one to avoid repetitions: https://github.com/RasaHQ/rasa/issues/5229